src/libstore/store-api.hh

```diff
@@ -445,7 +449,7 @@ public:
        sizes) of a set of paths.  If a path does not have substitute
        info, it's omitted from the resulting ‘infos’ map. */
     virtual void querySubstitutablePathInfos(const StorePathSet & paths,
-        SubstitutablePathInfos & infos) { return; };
+        SubstitutablePathInfos & infos, std::map<std::string, std::string> pathsCA = {}) { return; };
```

Comment:
Need a comment here what `pathsCA` is for.

Reply:
Also, if I understand it correctly, `paths` and `pathsCA` should be combined into a type like `std::map<StorePath, std::optional<std::string>>`.

Reply:
Yeah that would be better. Is it okay to make changes to store-api.hh like that? We'd have to replace old querySubstitutablePathInfos that may not care about ca.